setup/create_sqlite_db.py

```diff
@@ -0,0 +1,21 @@
+import sqlite3
```

Comment:
Move this to `mora.cli` and integrate it to our documented CLI using [Click](https://click.palletsprojects.com).